tkg/tkgctl/create_cluster.go

```diff
@@ -160,7 +160,7 @@ func (t *tkgctl) processManagementClusterInputFile(ir *InitRegionOptions) (bool,
 			return isInputFileClusterClassBased, err
 		}
 		if isInputFileClusterClassBased {
-			err = t.processClusterObjectForConfigurationVariables(clusterobj)
+			err = t.processClusterObjectForConfigurationVariables(clusterobj, ir.ClusterConfigFile)
```

Comment:
Since this is only invoked by mc creation, how to pass it to workload cluster creation?

Reply:
When users create a workload cluster with a legacy configuration file, cli terminate creation and generate a classy cluster manifest. Users need to run `tanzu cluster create --file <generated manifest file>`. So this is mainly for workload cluster creation.

Reply:
But why is this api only invoked in a mc creation api?

Reply:
It is invoked in both mc and wc creation. Please refer to line 186